views/partials/integrations-list.hbs

```diff
@@ -1,11 +1,20 @@
 <ul class="integrations-list">
     <li class="integration">
-        <a class="integration-content" href="{{pathTo 'handlebars'}}">
-            <h3 class="integration-name">Handlebars</h3>
-            <div class="integration-brand">
-                <img class="integration-mark" alt="Handlebars logo" src="/img/handlebars.svg">
-            </div>
-        </a>
+      <a class="integration-content" href="{{pathTo 'ember'}}">
```

Comment:
Because of the dust logo has nearly equal height, I put them on opposing sides to "balance" the logo row